src/include-codeblock.js

```diff
@@ -16,3 +18,17 @@ module.exports = {
         }
     }
 };
+
+try {
+    require.resolve("gitbook-plugin-ace");
```

Comment:
Is this line needed?

or need comments 

// step1 : check abc
// step2: check def